modules/ROOT/pages/actor-filter-archetype.adoc

```diff
@@ -55,16 +58,16 @@ A folder named _[your artifact id]_ is created, with your Bonita actor filter pr
 You can avoid the interactive mode by specifying all properties of your project directly in the command line, but by doing that you'll bypass the validation performed on the properties content.
 ====
 
-=== Actor filter developpment
+=== Actor filter development
 
 In this section we'll look into the different components of an actor filter project, and how you should use them to develop your filter.
 
 ==== Definition
 
 An actor filter is first defined by its *definition*.  It is an XML file located in _src/main/resources-filtered/[artifactId].def_ by default. +
-A definition defines the inputs of a filter. It can be seen as a black box. The definition explicits what will be passed to the filter. Then, implementations of this definition can be created, they just need to respect the inputs contract of the definition.
+A definition defines the inputs of an actor filter. It can be seen as a contract. Then, implementations of this definition can be created, they just need to respect the inputscontract of the definition.
```

Comment:
```suggestion
A definition defines the inputs of an actor filter. It can be seen as a contract. Then, implementations of this definition can be created, they just need to respect the inputs contract of the definition.
```